go-selinux/selinux_linux.go

```diff
@@ -548,37 +548,35 @@ func rangeStrToMLSRange(rangeStr string) (*mlsRange, error) {
 
 // bitsetToStr takes a category bitset and returns it in the
 // canonical selinux syntax
-func bitsetToStr(c *bitset.BitSet) string {
+func bitsetToStr(c *big.Int) string {
 	var str string
-	i, e := c.NextSet(0)
-	len := 0
-	for e {
-		if len == 0 {
+
+	i := int(c.TrailingZeroBits())
+	length := 0
+	for i < c.BitLen() {
+		if c.Bit(i) == 0 {
+			i++
+			continue
+		}
+		if length == 0 {
 			if str != "" {
 				str += ","
 			}
-			str += "c" + strconv.Itoa(int(i))
+			str += "c" + strconv.FormatInt(int64(i), 10)
```

Comment:
From the docs:

> Itoa is equivalent to FormatInt(int64(i), 10).

so why change this?

Reply:
no particular reason,  probably i deleted the old lines when rewrote and used my own tastes.

grep the strconv usage in package, seems iota and itoa are more used. let's keep the style.